kratos/sources/model_part.cpp

```diff
@@ -149,10 +149,9 @@ ModelPart::IndexType ModelPart::CreateSolutionStep()
 
 ModelPart::IndexType ModelPart::CloneSolutionStep()
 {
-    if (IsSubModelPart())
-        //Todo KRATOS_THROW_ERROR(std::logic_error, "Calling the method of the sub model part ", Name())
-        KRATOS_ERROR << "Calling the CloneSolutionStep method of the sub model part " << Name()
-                     << " please call the one of the parent modelpart : " << mpParentModelPart->Name() << std::endl;
```

Comment:
giving the name of the parent is wrong here, since this might also be a submodelpart